### ISSUE TITLE
[4.0] UX - Article table options multiple select filter fields lack of default value can confuse user

### ISSUE THREAD

**Issue by emuthomi** [joomla/joomla-cms#19882]
### Steps to reproduce the issue
If this issue has been reported elsewhere apologies in advance.

Go to Articles and click Table options, which is the new filter tool. You will see that some filter fields have blank hints, communicating no clue on what they are for.

![screenshot-localhost-2018-03-10-14-12-39-625](https://user-images.githubusercontent.com/3242390/37241789-2537ecba-2470-11e8-9841-06d0f6c04232.png)

Only after clicking do you see a dropdown of options:

![screenshot-localhost-2018-03-10-14-13-44-585](https://user-images.githubusercontent.com/3242390/37241798-625e198e-2470-11e8-98be-365f390f48ed.png)

Unfocusing the field or clicking away is when the hint appears on the field you had clicked:

![screenshot-localhost-2018-03-10-14-14-31-148](https://user-images.githubusercontent.com/3242390/37241816-9d990630-2470-11e8-97b5-e4c792843172.png)

Inspecting the HTML shows that the input field has a width of 0px which is the cause of the small bug.

![screenshot-localhost-2018-03-10-14-27-33-456](https://user-images.githubusercontent.com/3242390/37241839-e920c3ea-2470-11e8-945e-0250af52d5cf.png)

Comment the style and the hint appears in the field:

![screenshot-localhost-2018-03-10-14-31-18-209](https://user-images.githubusercontent.com/3242390/37241859-251ecce8-2471-11e8-9757-1cd2059c0d47.png)

Another Issue is that once you have selected some options, you cannot remove them by clicking the X 

![screenshot-localhost-2018-03-10-14-11-35-276](https://user-images.githubusercontent.com/3242390/37241875-63c86166-2471-11e8-99f2-63edef024baa.png)

### System information (as much as possible)

Joomla! 4.0.0-dev Development [ Amani ] 19-November-2017
Browser: Opera 51 and Firefox Quantum

**Comment by brianteeman** [comment]
We should probably have a label above each dropdown filter
That way it is clear before anything is selected AND afterwards

**Comment by Quy** [comment]
This is no longer reproducible.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19882">issues.joomla.org/tracker/joomla-cms/19882</a>.</sub>
